siriuspy/siriuspy/csdevice/timesys.py

```diff
@@ -48,10 +48,28 @@
     )
 
 
+class Constants:
```

Comment:
Can we use <code>Const</code> instead of <code>Constants</code>, as in other modules?